<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Offers with additional charges"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="29.765">
  <properties />
            <testcase name="test_VerifyValidOfferPaymentTypeCCAndDC" time="29.765" classname="Test.MerchantPanel.Offers.TestOffers">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  Test.AdminPanel.Payments.TransactionHelper.MakeGatewayAsStatusUp(TransactionHelper.java:968)
  Test.AdminPanel.Payments.TransactionHelper.VerifyIbiboCodePresentForMerchant(TransactionHelper.java:1087)
  Test.AdminPanel.Payments.TransactionHelper.DoTestTransaction(TransactionHelper.java:788)
  Test.AdminPanel.Payments.TransactionHelper.DoTestTransaction(TransactionHelper.java:764)
  Test.MerchantPanel.Offers.TestOffers.test_VerifyValidOfferPaymentTypeCCAndDC(TestOffers.java:297)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:483)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:46)
  org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:37)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:745)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
